absl/time/internal/cctz/testdata/zoneinfo/iso3166.tab

```diff
@@ -10,7 +10,7 @@
 #
 # 1.  ISO 3166-1 alpha-2 country code, current as of
 #     ISO 3166-1 N905 (2016-11-15).  See: Updates on ISO 3166-1
-#     http://isotc.iso.org/livelink/livelink/Open/16944257
+#     https://isotc.iso.org/livelink/livelink/Open/16944257
```

Comment:
This change needs to be reverted and submitted to https://github.com/google/cctz. (This is a common mistake. We should update our documentation.)

Reply:
The same goes for all files under absl/time/internal/cctz. Sorry about this.

Reply:
Alright, I've done this and will submit to google/cctz soon.